docs/api/system-preferences.md

```diff
@@ -380,6 +380,30 @@ You can use the `setAppLevelAppearance` API to set this value.
 Sets the appearance setting for your application, this should override the
 system default and override the value of `getEffectiveAppearance`.
 
+### `systemPreferences.isTouchIDAvailable()` _macOS_
+
+Returns `Boolean` - whether or not this device has the ability to use Touch ID.
+
+**NOTE:** This API is only available on macOS Sierra 10.12.2 or newer.
+
+### `systemPreferences.promptTouchID(reason)` _macOS_
+
+* `reason` String - The reason you are asking for Touch ID authentication
+
+Returns `Promise<Boolean>` - whether ot not the user has successfully authenticated with Touch ID.
```

Comment:
_(nit)_ s/ot/or/

_(non-nit)_ This doc implies that the app dev should test the boolean to see if they succeeded, but the boolean is always True. IMO this should be a `Promise<void>`